src/darwin/Framework/CHIP/CHIPP256KeypairBridge.h

```diff
@@ -49,7 +49,7 @@ class CHIPP256KeypairBridge : public chip::Crypto::P256KeypairBase
     const chip::Crypto::P256PublicKey & Pubkey() const override { return mPubkey; };
 
 private:
-    id<CHIPKeypair> mKeypair;
+    id<CHIPKeypair> _Nullable mKeypair;
```

Comment:
Again, does `nullable id<CHIPKeypair> mKeypair;` work?

Reply:
It failed to compile (was my first try as readable option). Also placement is picky: nullable has to be before id it seems.